components/table/table-wrapper.js

```diff
@@ -575,12 +576,27 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 	}
 
 	async _handleTableChange() {
-		await new Promise(resolve => requestAnimationFrame(resolve));
+		const debounceDelay = 300;
+		const update = async() => {
+			await new Promise(resolve => requestAnimationFrame(resolve));
 
-		this._updateItemShowingCount();
-		this._applyClassNames();
-		this._syncColumnWidths();
-		this._updateStickyTops();
+			this._updateItemShowingCount();
+			this._applyClassNames();
+			this._syncColumnWidths();
+			this._updateStickyTops();
+			this.dispatchEvent(new CustomEvent('d2l-table-changed', { bubbles: true, composed: false }));
+			this._recentTableChange = true;
```

Comment:
We'd typically initialize local variables like this in the constructor just to make their existence explicitly clear, if if they're `false` or `null` initially.